components/cluster/command/import.go

```diff
@@ -124,7 +124,7 @@ func newImportCmd() *cobra.Command {
 			}
 
 			// copy config files form deployment servers
-			if err = ansible.ImportConfig(clsName, clsMeta, gOpt.SSHTimeout, gOpt.NativeSSH); err != nil {
+			if err = ansible.ImportConfig(clsName, clsMeta, gOpt.SSHTimeout, gOpt.SSHType); err != nil {
```

Comment:
https://github.com/pingcap/tiup/pull/699/files#diff-59e7c700e2fb6e772c3f01fdbd134d51R46

The `ImportConfig` doesn't specify the default ssh type, is it right?

Reply:
There is no default ssh type when you are importing a cluster since tidb-ansible doesn't record that

Reply:
If the user doesn't specify the `--ssh`, the import process will abort. I mean there is should be a default value for `--ssh`.

Reply:
> the import process will abort

I don't think so, "" is equals "builtin" for executor.New.